announcementsAPI.json

```diff
@@ -389,6 +399,48 @@
         },
         "additionalProperties": false
       },
+      "PetResponse": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "format": "uuid",
+            "nullable": false
+          },
+          "name": {
+            "type": "string",
+            "nullable": true
+          },
+          "sex":{
+            "$ref": "#/components/schemas/Sex"
+          },
+          "species": {
+            "type": "string",
+            "nullable": true
+          },
+          "breed": {
+            "type": "string",
+            "nullable": true
+          },
+          "birthday": {
+            "type": "string",
+            "format": "date-time"
+          },
+          "description": {
+            "type": "string",
+            "nullable": true
+          },
+          "photo": {
```

Comment:
You mean photoUrl? If so, i don't undestand why "byte" format? Unfortunately swagger didn't include response data structures in my export but we had something like this: 

{
    "id": "",
    "shelter": null,
    "name": "",
    "species": "",
    "breed": "",
    "birthday": "",
    "description": "",
    "photoUrl": ""
}

Reply:
I branched out of `master` before the changes to pet photos, so I haven't changed that yet. It will be fixed as soon as the other PR is merged and I merge those changes into this branch.